src/domain/SipLine.js

```diff
@@ -6,51 +6,76 @@ const availableCodecs = ['vp8', 'vp9', 'h264'];
 
 type SipLineResponse = {
   id: number,
+  uuid: string,
   tenant_uuid: string,
   username: string,
   secret: string,
   type: string,
   host: string,
-  options: Array<Array<string>>,
+  options: ?Array<Array<string>>,
+  auth_section_options: ?Array<Array<string>>,
+  endpoint_section_options: ?Array<Array<string>>,
   links: Array<Object>,
   trunk: ?string,
   line: Endpoint,
 };
 
 type SipLineArguments = {
   id: number,
+  uuid: string,
   tenantUuid: string,
   username: string,
   secret: string,
   type: string,
   host: string,
-  options: Array<Array<string>>,
+  options: ?Array<Array<string>>,
+  endpointSectionOptions: ?Array<Array<string>>,
   links: Array<Object>,
   trunk: ?string,
   line: Endpoint,
 };
 
 export default class SipLine {
-  id: number;
+  id: any;
+  uuid: string;
   tenantUuid: string;
   username: string;
   secret: string;
   type: string;
   host: string;
-  options: Array<Array<string>>;
+  options: ?Array<Array<string>>;
+  endpointSectionOptions: ?Array<Array<string>>;
   links: Array<Object>;
   trunk: ?string;
   line: Endpoint;
 
   static parse(plain: SipLineResponse): SipLine {
+    let { username, secret, host } = plain;
+
+    // Since 20.13 engine so options are now in section
+    if (plain.auth_section_options) {
+      // $FlowFixMe
+      const findOption = (name: string) => plain.auth_section_options.find(option => option[0] === name);
+
+      const usernameOption = findOption('username');
+      const secretOption = findOption('password');
+      const hostOption = findOption('media_address');
```

Comment:
Nice, great refactor :) But media_address is in `endpoint_section_options` not `auth_section_options` :(.

Reply:
Why do we need media_address?

Reply:
To build a sip URI. It's maybe not the right option to use, but we need a host and it was the only option I had in the options to looks like the right host. Do you know the right option to use ?

Reply:
huuuuu :) We need to discuss about that, i don't understand why we are using this information for that. The user fill this information at the login.

Reply:
This information is not used anywhere in our app.

Reply:
So why do we need to update this code? For me there is no sense to have that in the domain.

Reply:
To avoid breaking change. Maybe someone on this planet is using this.

Reply:
I don't understand why someone use that...

Reply:
Me neither, but it's something we expose and we should expose the correct value

Reply:
ok but what is media_address in the new API, it doesn't exist for me.

Reply:
Did you used the merged view params? Otherwise you wont see it `/api/confd/1.1/endpoints/sip/{sip_uuid}?view=merge`